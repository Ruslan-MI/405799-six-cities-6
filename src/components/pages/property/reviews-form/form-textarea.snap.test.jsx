import React from "react";
import {
  render,
} from "@testing-library/react";
import FormTextarea from "./form-textarea";

it(`Form textarea should render correctly`, () => {
  const textareaMockValue = `Mock value`;

  const onChange = jest.fn();

  const {
    container,
  } = render(<FormTextarea value={textareaMockValue} isDisabled onChange={onChange} />);

  expect(container).toMatchSnapshot();
});
