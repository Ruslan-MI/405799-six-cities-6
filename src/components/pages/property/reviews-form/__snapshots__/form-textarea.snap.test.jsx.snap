// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form textarea should render correctly 1`] = `
<div>
  <textarea
    class="reviews__textarea form__textarea"
    disabled=""
    id="review"
    maxlength="300"
    minlength="50"
    name="review"
    placeholder="Tell how was your stay, what you like and what can be improved"
  >
    Mock value
  </textarea>
</div>
`;
